tests/demo.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from peerdid.peer_did_creator import create_peer_did_numalgo_0, create_peer_did_numalgo_2
+from peerdid.peer_did_resolver import resolve_peer_did
+from peerdid.storage import FileStorage
+from peerdid.types import PublicKey, KeyType
+
+
+def test_create_save_resolve_peer_did():
+    encryption_keys = [PublicKey(encoded_value="Ez6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH",
+                                 type=KeyType.X25519)]
+    signing_keys = [PublicKey(
+        encoded_value="zXwpBnMdCm1cLmKuzgESn29nqnonp1ioqrQMRHNsmjMyppzx8xB2pv7cw8q1PdDacSrdWE3dtB9f7Nxk886mdzNFoPtY",
+        type=KeyType.X25519)]
```

Comment:
X25519 can be used for keyAgreement (encryption) only.
Ed25519 should be used for signatures.
Probably it makes sense to separate PublicKey class and type enum into two: PublicKeyAuthentication, PublicKeyKeyAgreement